explainer.md

```diff
@@ -298,6 +299,14 @@ This function is provided for debugging / logging purposes, and may have
 significant performance implications.
 
 
+## Maximum bucket count
+
+A user agent can specify the maximum number of buckets allowed for an origin.
```

Comment:
I do think the spec should (consider) requiring user agents to support at least a certain number of buckets. The number of buckets supported could fundamentally change the architecture of a web site. Also for user agents thinking about implementing this API it would be good to give some indications for the expected values for this value. An implementation supporting 2 buckets is likely quite different from an implementation supporting 1000 buckets.